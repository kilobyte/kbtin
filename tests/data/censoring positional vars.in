#act {%1} {#showme {% %1};#showme {$ $1}}
#gag {%1}
#run a //selfpipe
#de 2 #end
#
a;b
{{{{
{}{}{
{}}}
{}{}{}
{{{}}}
