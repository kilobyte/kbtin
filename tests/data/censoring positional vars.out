#All text is now sent 'as is'.
% a;b
$ ab
% {{{{
$ ((((
% {}{}{
$ {}{}(
% {}}}
$ {}))
% {}{}{}
$ {}{}{}
% {{{}}}
$ {{{}}}

#SESSION 'a' DIED.
